src/App.tsx

```diff
@@ -1,14 +1,32 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { actions as todosActions } from './features/todos';
+
+const ERROR_MESSAGE = 'Failed to load the todos, try again later';
 
 export const App: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const currentTodo = useAppSelector(state => state.currentTodo);
+
+  const [error, setError] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getTodos()
+      .then(todosFromServer => dispatch(todosActions.load(todosFromServer)))
+      .catch(() => setError(ERROR_MESSAGE))
+      .finally(() => setIsLoading(false));
+  }, []);
```

Comment:
Try to fix the linter error.

Error: React Hook useEffect has a missing dependency: 'dispatch'. Either include it or remove the dependency array